integration/js/SipCallTest.js

```diff
@@ -34,7 +34,7 @@ class SipCallTest extends SdkBaseTest {
     const sipCallClient = new SipCallClient(this.payload.sipTestAssetPath, this.payload.resultPath);
     const sipCall = sipCallClient.call(this.payload.voiceConnectorId, this.sipUri);
 
-    await test_window.runCommands(async () => await await RosterCheck.executeStep(this, session, 2));
+    await test_window.runCommands(async () => await await RosterCheck.executeStep(this, session, 2, new RosterCheckConfig(60, 500)));
```

Comment:
Will 30 second be enough? I thought this could take 35s before the SIP attendee shows up.